protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3143,29 +3143,26 @@ message OrderInfo {
   vega.Side side = 1;
   // Price for the order. The price is an integer, for example `123456` is a correctly
```

Comment:
If this is unsigned, we should still keep that detail in here. Not sure what to do about the fact that it's a string but that it's returned as a number - how can that be described clearly?

Reply:
I think we removed the "expressed as a string" in most places because the type definition on the field dictates that it should be expressed as a string anyway, but we kept the description of how the integer should be expressed to indicate the data in the string should be an integer.

Reply:
Happy to change it if you think it's not clear, but we would need to do it in a few other places too.